astropy/coordinates/representation/tests/test_representation_arithmetic.py

```diff
@@ -3,20 +3,18 @@
 import functools
```

Comment:
But if we move tests, we should remove `representation` from the test file name. So, `test_basics` above, `test_arithmetic` here.